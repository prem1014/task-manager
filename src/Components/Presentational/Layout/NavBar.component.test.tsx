import React from 'react';
import Enzyme, { shallow, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import NavBar from './NavBar.component';

describe('Navbar component test suite', () => {
    Enzyme.configure({ adapter: new Adapter() });
    let componentWrapper: any;

    beforeEach( () => {
        componentWrapper = shallow(<NavBar />);
    });

    it('Navbar component snapshot test', () => {
        expect(componentWrapper).toMatchSnapshot();
    });
});