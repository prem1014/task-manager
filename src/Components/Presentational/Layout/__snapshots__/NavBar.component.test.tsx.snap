// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component test suite Navbar component snapshot test 1`] = `ShallowWrapper {}`;
